packages/aws-cdk/test/api/hotswap/cloudwatch-logs.test.ts

```diff
@@ -0,0 +1,241 @@
+import { CloudWatchLogEventMonitor } from '../../../lib/api/monitor/logs-monitor';
```

Comment:
This file is probably in the wrong directory now.  `test/api/logs/` would be my choice.